lib/data.dart

```diff
@@ -98,12 +91,9 @@ abstract class Homework implements Built<Homework, HomeworkBuilder> {
   String? get gradeFormatted;
 
   String? get grade;
-  // It's always false for tests :(. I only saw a warning for a grade entry with
-  // a 'null' grade.
-  @BuiltValueField(wireName: "warning")
-  bool get warningServerSaid;
+
   // Heuristics to still show a warning when it would make sense
```

Comment:
Can you update this comment? It's no longer just a heuristic/guess, so this line could be removed entirely.